src/acl_printf.cpp

```diff
@@ -960,13 +981,14 @@ static void l_dump_printf_buffer(cl_event event, cl_kernel kernel,
 #ifdef DEBUG
   printf("exiting acl_dump_buffer...\n");
 #endif
+  return dumped_buffer_size;
 }
 
 //
 // Schedule enqueue read buffer to read printf buffer
 // The activation ID is the device op ID.
 void acl_schedule_printf_buffer_pickup(int activation_id, int size,
-                                       int stalled) {
+                                       int debug_dump_printf) {
```

Comment:
Does this HAL API also need to be updated?
https://github.com/intel/fpga-runtime-for-opencl/blob/67f7f880f7af5af0c1797c4dd2a70b4e27ba85ca/include/acl_hal.h#L65-L66